.github/workflows/reusable-publish-vsix.yml

```
@@ -0,0 +1,99 @@
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: 'Release tag to publish from, e.g. v5.1.0'
+        required: true
+        type: string
+      vscode:
+        description: 'Publish to VS Code Marketplace'
+        required: true
+        type: boolean
+        default: true
```

actionlint (1.7.12)

[error] input "vscode" of workflow_call event has the default value "true", but it is also required. if an input is marked as required, its default value will never be used

(events)